### ISSUE TITLE
feature: allow extra hashcat command parameters in client config

### ISSUE THREAD

**Issue by roycewilliams** [hashtopolis/server#895]
In a loose collaboration scenario (lots of independent agents run by different people), it would be super useful if the user could set extra parameters on the client side. in other words, if `config.json` could have an "extra_parameters" clause, where people could add things like " `-w 4` " or " `--backend-ignore-hip` " or " `-D 1` ", etc. This would allow people to follow a FAQ or orientation guide to adjust their own settings based on the errors they see, without having to wait for server-side intervention.

**Comment by s3inlc** [comment]
Just a side note on the use-case. Exact for such things, there is the possibility to have users which are just able to see their own agents, to manage an agent on their own without a server admin. Of course I understand that it makes it slightly more complicated as then each user needs an account, but having this additional feature also creates some sort of duplicate features.

**Comment by roycewilliams** [comment]
Yes, totally understood - but in large loose-collaboration scenarios, the more human-intervention steps that we can push to the client (and a tech-savvy user on the client side) the better (see my comment with more detail here: https://github.com/hashtopolis/server/issues/898#issuecomment-1494346537)

**Comment by s3inlc** [comment]
I see, though such collaboration settings are not that common, it may be useful.
Though, I would propose that such a feature would need to be enabled in the server config and being turned off by default, as in typical setups, the agent should not have much power and the server is the central management place.
